config/proto/service.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+
+service Hermes {
+
+  // Start monitoring a new storage system.
+  rpc StartMonitoringStorageSystem(HermesProbeRequest) returns (HermesProbeResponse) {}
+
+  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
+
+  // Lists the storage systems being monitored at the moment.
+  rpc ListMonitoredStorageSystems(ListMonitoredSystemsRequest) returns (ListMonitoredSystemsResponse) {}
+}
+
+message HermesProbeRequest {
+  required HermesProbeDef probe_confg = 1;
+}
+
+message HermesProbeResponse {}
+
+message StopMonitoringSystemRequest {
+ required Target target = 1;
```

Comment:
I gather you are importing this from the targets.proto config file, make sure to prepend the 'Target' message type with its package path.

Ditto for the other mentions below.

Reply:
I have addressed this in a new commit.